public/viewer/viewer.js

```diff
@@ -8402,7 +8402,7 @@ function webViewerInitialized() {
   var queryString = document.location.search.substring(1);
   var params = parseQueryString(queryString);
   var file = 'file' in params ? params.file : DEFAULT_URL;
-  validateFileURL(file);
+  //validateFileURL(file);
```

Comment:
It's not a big deal, it's for when you want to limit loading PDFs from your own domain. As we don't want to do that (PDFs come from Canvas) and we don't allow user input, it's not that big of a problem imho